AutoSteamApp/Helpers/StaticHelpers.cs

```diff
@@ -0,0 +1,179 @@
+            Log.Message("\tRare Reward Success Rate: " + ConfigurationReader.RareSuccessRate.ToString("#.##"));
+            Log.Message("\tMaximum Wait for Determining Slot: " + ConfigurationReader.MaxTimeSlotNumberSeconds);
+            Log.Message("");
+        }
+
+
+        /// <summary>
+        /// Sets the proper log methods depending on the configuration found in the .config file
+        /// </summary>
+        public static void SetLogger()
+        {
+            // Initialize the different log types
+            Log.LogTypes LoggingTypes = Log.LogTypes.Message | Log.LogTypes.Warning;
+
+            // If we're in debug mode, heighten the logging which takes place
+            if (ConfigurationReader.IsDebug)
+            {
+                LoggingTypes |= Log.LogTypes.Debug | Log.LogTypes.Exception | Log.LogTypes.Error;
+
+                // Check if we need to write logs to a file
+                string logFile = ConfigurationReader.LogFile;
+                if (!string.IsNullOrEmpty(logFile))
+                {
+                    // Try to create a stream using the log file
+                    try
+                    {
+                        FileStream logStream = File.Create(logFile);
+                        Log.SetStream(logStream, true, LoggingTypes);
+                        return;
+                    }
+                    // In the event of failure, log to console instead
+                    catch (Exception e)
+                    {
+                        Log.SetStream(Console.OpenStandardOutput(), false, LoggingTypes);
+                        Log.Exception(e, logFile);
+                        return;
+                    }
+                }
+                // If no log file is specified, log to console instead
+                Log.SetStream(Console.OpenStandardOutput(), false, LoggingTypes);
+                //Log.Warning("No log file specified. All logs written to console.");
+                return;
+            }
+            // If not in debug mode, write messages to console
+            Log.SetStream(Console.OpenStandardOutput(), false, LoggingTypes);
+        }
+
+        /// <summary>
+        /// Sets the console title of the app
+        /// </summary>
+        public static void SetConsoleTitle()
+        {
+            Version ver = ConfigurationReader.ApplicationVersion;
+            string title = "Auto Steamworks Version:" + ver.Major + "." + ver.Minor;
+            title += "        ";
+            title += "Supported MHW:IB Version: " + MHWMemoryValues.SupportedGameVersion;
+            Console.Title = title;
+        }
+
+        /// <summary>
+        /// Converts the byte sequence to an array of Virtual Key Codes using the index of the key as the order in which to press
+        /// <param name="sequence"></param>
+        /// <returns></returns>
+        public static VirtualKeyCode[] KeyCodeSequenceFromBytes(byte[] sequence)
```

Comment:
Naming is `neutral` meaning it doesn't tell you exactly what this method does - a good fit could be `GetKeyCodeSequenceFromBytes`